File: Solutions/JavaScript/pattern013.js
let pattern = ''
let n = 5
let x
let y

for (let i=1; i<=n; i++) {
    x = i
    y = n - i + 1
    for (let j = 1; j<=n; j++ ) {
        if (j % 2 == 1) {
            pattern += x.toString().padStart(3, ' ')
        } else {
            pattern += y.toString().padStart(3, ' ')
        }
        x = x + n
        y = y + n
    }
    pattern += '\n'
}
console.log(pattern)

File: Solutions/C/pattern013.c
// Pattern Programs - 012 [C]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <stdio.h>

int main(void)
{
    int n = 5;
    int i, j;
    int x, y;

    for (i = 1; i <= n; i++)
    {
        x = i;
        y = n - i + 1;
        for (j = 1; j <= n; j++)
        {
            if (j % 2 == 1)
            {
                printf("%2d ", x);
            }
            else
            {
                printf("%2d ", y);
            }
            x = x + n;
            y = y + n;
        }
        printf("\n");
    }
    return 0;
}